src/style.css

```diff
@@ -13,7 +13,7 @@
 }
 #right-panel{
     width: 35%;
-    float: right;
+    float: right !important; 
```

Comment:
## Overuse of "!important"
- kindly minimize the usage of "!important"
- it overrides just about everything, even inline styles, and messes in a less-than-obvious way with the "cascade" of style rules that gives CSS its name